modules/hls.xml

```diff
@@ -0,0 +1,103 @@
+<!--
+  ~ Copyright (C) 2022 Xibo Signage Ltd
+  ~
+  ~ Xibo - Digital Signage - http://www.xibo.org.uk
+  ~
+  ~ This file is part of Xibo.
+  ~
+  ~ Xibo is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU Affero General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ any later version.
+  ~
+  ~ Xibo is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU Affero General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Affero General Public License
+  ~ along with Xibo.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+<module>
+    <id>core-hls</id>
+    <name>HLS</name>
+    <author>Core</author>
+    <description>A module for displaying HLS video streams</description>
+    <class></class>
+    <type>hls</type>
+    <dataType></dataType>
+    <schemaVersion>1</schemaVersion>
+    <assignable>1</assignable>
+    <regionSpecific>1</regionSpecific>
+    <renderAs>html</renderAs>
+    <defaultDuration>60</defaultDuration>
+    <settings></settings>
+    <properties>
+        <property id="uri" type="input">
+            <title>Video Path</title>
+            <helpText>A URL to the HLS video stream. Requires Player running Windows 8.1 or later, or Android 6 or later. Earlier Android devices may play HLS via the LocalVideo widget.</helpText>
+        </property>
+        <property id="mute" type="checkbox">
+            <title>Mute?</title>
+            <helpText>Should the video be muted?</helpText>
+            <default>0</default>
+        </property>
+    </properties>
+    <preview></preview>
+    <stencil>
+        <twig>
+            <![CDATA[
+                <video id='video' poster='data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVQYV2NgYAAAAAMAAWgmWQ0AAAAASUVORK5CYII=' {% if mute == 0 %} muted {% endif %} ></video>
```

Comment:
We can use this opportunity to make better formatted HTML, in this case:
 - `"` instead of `'`
 - no spaces in the final output